grammar.js

```diff
@@ -264,7 +265,7 @@ const haxe_grammar = {
     keyword: ($) => prec.right(choice(...keywords)),
     identifier: ($) => /[a-zA-Z_]+[a-zA-Z0-9]*/,
     // Hidden Nodes in tree.
-    _semicolon: ($) => ';',
+    _semicolon: ($) => $._lookback_semicolon,
```

Comment:
In the few places left where `_semicolon` is still used instead of `_lookback_semicolon`, there is no need for this extra logic (package, import and using statements). So it is still useful to distinguish from the regular `_semicolon` token.